src/components/Profile/VpDelegatorsBox.tsx

```diff
@@ -15,24 +15,27 @@ import './VpDelegatorsBox.css'
 
 interface Props {
   address: string | null
+  userAddress: string | null
   delegation: DelegationResult
   delegationState: AsyncStateResultState<DelegationResult, DelegationResult>
   scores: DetailedScores
   isLoadingScores: boolean
-  vpDistribution: VpDistribution | null
 }
 
 export default function VpDelegatorsBox({
   address,
+  userAddress,
   delegation,
   delegationState,
   scores,
   isLoadingScores,
-  vpDistribution,
 }: Props) {
   const t = useFormatMessage()
-  const [userAddress] = useAuthContext()
   const isLoggedUserProfile = isSameAddress(userAddress, address)
+  const { vpDistribution: loggedUserVpDistribution, isLoadingVpDistribution } = useVotingPowerInformation(
+    !isLoggedUserProfile ? userAddress : undefined
```

Comment:
wouldn't it be better to update the hook to also accept null values and have the same behavior as undefined?

Reply:
Hmm.. I don't see how that would change behavior in this case.
What I intend here is: If the profile we are looking at is not the logged user's, then load the user vp information. 
If not, the hook receives undefined and avoids calling the APIs.
 But `userAddress` could be a valid value, and still we would want to send `undefined` to the hook.

Reply:
aaah okay, I wasn't contemplating the last case you described. Okay then!